core/src/test/java/test/failedreporter/issue2521/DataProviderWithFactoryMultiFailedReporterTest.java

```diff
@@ -0,0 +1,54 @@
+package test.failedreporter.issue2521;
+
+import java.io.File;
+import java.util.List;
+import org.testng.Assert;
+import org.testng.TestNG;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.collections.Lists;
+import test.SimpleBaseTest;
+
+public class DataProviderWithFactoryMultiFailedReporterTest extends SimpleBaseTest {
```

Comment:
Can you please add this test to `src/test/resources/testng.xml` so that it gets included for every run ?

Reply:
oh, I'm sorry. I have done.